robot-server/robot_server/runs/router/base_router.py

```diff
@@ -185,14 +185,19 @@ async def create_run(
     },
 )
 async def get_runs(
+    pageLength: Optional[int] = Query(
+        None,
+        description="The maximum number of runs in the list to return ordered by oldest to newest.",
+    ),
```

Comment:
We might want to explicitly document that if you omit `pageLength`, it will return *all* runs. The current Opentrons desktop app depends on this, and it's unlike `GET /runs/{id}/commands`.

We should also document the behavior that we discussed in-person yesterday: it returns the *most recent* `pageLength` runs, not the oldest ones.

```suggestion
    pageLength: Optional[int] = Query(
        None,
        description=(
            "The maximum number of runs to return."
            " If this is less than the total number of runs,"
            " the most-recently created runs will be returned."
            " If this is omitted or `null`, all runs will be returned."
        ),
    ),
```

Reply:
In that docstring, I'm presuming that JS people expect to be able to pass the string "null" as the query param value, but I could be wrong! I'm curious what other people think.